astropy/coordinates/representation/base.py

```diff
@@ -244,23 +262,18 @@ def __init__(self, *args, **kwargs):
         if any(hasattr(attr, "mask") for attr in attrs):
             self._ensure_masked()
 
+    # -----------------------------------------------------------------
```

Comment:
Why this extra line?

Reply:
legacy of the `.name` and `.get_name` stuff. Removing.

Reply:
Done.